portal.go

```diff
@@ -2181,6 +2182,21 @@ func (portal *Portal) HandleWhatsAppInvite(source *User, senderJID *types.JID, j
 	return
 }
 
+func (portal *Portal) HandleWhatsAppDeleteChat() {
+	matrixMembers, err := portal.GetMatrixUsers()
+	if err != nil {
+		portal.log.Errorln("Couldn't get Matrix users to figure out if deleteChat should be handled!")
```

Comment:
Similar log fix as in the other PR

```suggestion
		portal.log.Errorln("Failed to get Matrix users to see if DeleteChat should be handled:", err)
```